packages/url_launcher/url_launcher_web/example/pubspec.yaml

```diff
@@ -1,22 +1,22 @@
 name: regular_integration_tests
 publish_to: none
 
-environment:
-  sdk: ">=2.10.0-56.0.dev <3.0.0"
-
 dependencies:
   flutter:
     sdk: flutter
 
 dev_dependencies:
+  build_runner: ^1.10.0
+  mockito: ^5.0.0-nullsafety.5
+  url_launcher_web:
+    path: ../
   flutter_driver:
     sdk: flutter
   flutter_test:
     sdk: flutter
-  http: ^0.12.2
-  mockito: ^4.1.1
-  url_launcher_web:
-    path: ../
   integration_test:
-    path: ../../../integration_test
+    sdk: flutter
 
+environment:
```

Comment:
we generally leave this at the top – why the change?

Reply:
WAT? I've always seen this at the bottom of every pubspec I've touched so far!

(It looks weird at the top to me! 😅)

Reply:
I suspect at some point there was either a Flutter template that had it this way, or a plugin here ended up with it that way, and it's persisted/spread within the repo.

Reply:
(I would 100% approve a mass PR that moves them all to match the template, BTW, if anyone feels like doing that. I may do it at some point if not.)